rotkehlchen/tests/unit/accounting/evm/test_balancer.py

```diff
@@ -0,0 +1,219 @@
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_1_SECS,
+            asset=Asset('eip155:1/erc20:0xe2f2a5C287993345a840Db3B0845fbC70f5935a5'),
+            free_amount=ZERO,
+            taxable_amount=FVal('131578.947368421052491563'),
+            price=Price(ONE),
+            pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+            cost_basis=None,
+            index=0,
+            extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+        ),
+        ProcessedAccountingEvent(
+            type=AccountingEventType.TRANSACTION_EVENT,
+            notes='Deposit 131421.703252 USDC to a Balancer v1 pool',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_1_SECS,
+            asset=A_USDC,
+            free_amount=ZERO,
+            taxable_amount=FVal('131421.703252'),
+            price=Price(ONE),
+            pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+            cost_basis=None,
+            index=1,
+            extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+        ),
+        ProcessedAccountingEvent(
+            type=AccountingEventType.TRANSACTION_EVENT,
+            notes='Refunded 6578.947368421052624578 mUSD after depositing in Balancer V1 pool',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_1_SECS,
+            asset=Asset('eip155:1/erc20:0xe2f2a5C287993345a840Db3B0845fbC70f5935a5'),
+            free_amount=ZERO,
+            taxable_amount=FVal('6578.947368421052624578'),
+            price=Price(ONE),
+            pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+            cost_basis=None,
+            index=2,
+            extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+        ),
+        ProcessedAccountingEvent(
+            type=AccountingEventType.TRANSACTION_EVENT,
+            notes='Refunded 6571.085163 USDC after depositing in Balancer V1 pool',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_1_SECS,
+            asset=A_USDC,
+            free_amount=ZERO,
+            taxable_amount=FVal('6571.085163'),
+            price=Price(ONE),
+            pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
+            cost_basis=None,
+            index=3,
+            extra_data={'tx_hash': EVM_HASH.hex()},  # pylint: disable=no-member
+        ),
+        ProcessedAccountingEvent(
+            type=AccountingEventType.TRANSACTION_EVENT,
+            notes='Receive 1675.495956074927519908 BPT from a Balancer v1 pool',
+            location=Location.ETHEREUM,
+            timestamp=TIMESTAMP_1_SECS,
+            asset=Asset('eip155:1/erc20:0x72Cd8f4504941Bf8c5a21d1Fd83A96499FD71d2C'),
+            free_amount=FVal('1675.495956074927519908'),
+            taxable_amount=ZERO,
+            price=Price(ONE),
+            pnl=PNL(taxable=ZERO, free=ZERO),  # Deposits are not taxable
```

Comment:
Withdrawals are not taxable as above.

But then .... comes the question. What about profit or loss. You can withdraw more/less than you deposited in which case profit or loss is realized. This needs to be accounted for.

Reply:
This is the event for receiving the pool token, not the withdrawal

Reply:
This is the case of only a deposit where you get a refund. For the case of withdrawing a token as we said will be handled in a different PR only with that change

Reply:
Make sure that you know who is assigned to do it and when. Let's please not "forget that we had to do that"

Reply:
You have not answered this @yabirgb

Reply:
There was nothing to answer. I got note of it. Can add an emoji next time

Reply:
1. Emoji
2. Either make an issue or a sticky note or a pin in discord. Something to remind you, otherwise you **will** forget